lib/lib.js

```diff
@@ -144,35 +159,42 @@ export function formatDate(d) {
 //     children: { [string]: PageTree },
 //   }
 
-export function buildPageTree(path, type = "", content = false) {
+export function buildPageTree(path, ordering = "", content = false) {
+  // get metadata from _index.md file in each folder. This is where page ordering (weight) is stored.
```

Comment:
Not true, it just gives us the parent directory’s metadata, weight is stored in each page’s metadata.

Reply:
will update